### PR TITLE
feature/BOUENO-19: Moving options to advanced section and downgrading node and npm in backend

### PR DESCRIPTION
**Pull request by matsovevada**

### CODE REVIEW

File: code/build.gradle [hunk 1]
```diff
@@ -188,10 +188,10 @@ repositories {
 
 node {
     // Version of node to use.
-    version = '18.17.1'
+    version = '16.15.1'
```
Comment (SkylineTardis) [inline code/build.gradle:191]: I commit skriver du upgrade, men du mener egentlig downgrade?

**Review by SkylineTardis — APPROVED** [review]

**Comment by Roeland-Bouvet** [comment]
Merge conflicts... :suspect:

File: code/build.gradle (hunk 1 above)
Comment (matsovevada) [reply code/build.gradle:191]: Det var upgrade til å starte med, så downgradet backend og frontend for at jitpack skulle kunne bygge :)

**Comment by matsovevada** [comment]
> Merge conflicts... :suspect:

Fixed @Roeland-Bouvet

**Review by tunestam — APPROVED** [review]

File: code/src/main/resources/site/i18n/phrases.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Admin translations
 site.active = Active
-site.active-help = Activate to display the inquiry box on your website. When a visitor closes the inquiry box or submits the inquiry form, their browser will remember this and prevent it from displaying again unless the identifier has changed in the meantime.
```
Comment (Roeland-Bouvet) [inline code/src/main/resources/site/i18n/phrases.properties:3]: Did this get removed by accident during the merge conflict resolve? :)

Comment (matsovevada) [reply code/src/main/resources/site/i18n/phrases.properties:3]: Yea, looks like it... Fixed now :)

**Review by Roeland-Bouvet — APPROVED** [review]

**Pull request merged**